docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+volumes:
+  node_modules:
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile.localdev
+    extra_hosts:
+      - 'host.docker.internal:host-gateway'
+    command: poetry run python manage.py runserver 0.0.0.0:${PORT:-8000}
+    volumes:
+      - .:/app:delegated,rw
+      - node_modules:/app/node_modules/
+    ports:
+      - '${PORT:-8000}:${PORT:-8000}'
+    environment:
+      SECRET_KEY: $SECRET_KEY
```

Comment:
Why are we duplicating these to pass them through? I'd recommend either using dummy values, or having `docker-compose` read from a `.env` file.